docs/publish-module/ios-specific-publish-flows/app-information-app-store.md

```diff
@@ -0,0 +1,35 @@
+---
+title: App Information from App Store
+metaTitle: App Information from App Store
+metaDescription: App Information from App Store
+sidebar_position: 1
+---
+import Screenshot from '@site/src/components/Screenshot';
+
+# App Information from App Store
+
+This step allows you to see app version information from TestFlight and the App Store on the same screen, along with the version you want to submit. When the step runs, your latest version information in TestFlight and App Store will be displayed as follows.
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE2917-infoDetail.png' />
+
+:::caution
+Make sure the [**App Store Connect API Key**](https://docs.appcircle.io/account/adding-an-app-store-connect-api-key#linking-appcircle-with-app-store-connect) is added to Appcircle and selected.
```

Comment:
Selected from where, it is not obvious. Can you give more detail about Publish settings or add a screenshot?